src/go/k8s/README.md

```diff
@@ -34,12 +35,35 @@ export KUBECONFIG=your/path/to/kubeconfig.yaml
 kind create cluster --config kind.yaml
 ```
 
-You can simply deploy the Redpanda operator by running the following command
+In order to have validating webhook the cert manager needs to be
+installed. You can do this by running the following command:
+
+```
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.yaml
```

Comment:
let's just link to their documentation here instead https://cert-manager.io/docs/installation/

Reply:
Ok, changed.